gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/task/KubernetesBuildTaskTest.java

```diff
@@ -56,7 +56,7 @@ class KubernetesBuildTaskTest {
   private boolean isBuildError;
 
   @BeforeEach
-  void setUp() throws IOException {
+  void setUp() {
```

Comment:
@Adityaturkhade : You also need to remove `import java.io.IOException;` now since that isn't used in the class anymore

Reply:
Just updated as per comment